tests/foreman/cli/test_remoteexecution.py

```diff
@@ -269,15 +286,15 @@ def test_positive_run_default_job_template_multiple_hosts_by_ip(
     @pytest.mark.skipif(
         (not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url'
     )
-    def test_positive_install_multiple_packages_with_a_job_by_ip(
+    def test_positive_install_remove_multiple_packages_with_a_job(
```

Comment:
Since you removed the `_by_ip` nonsense here, how about doing it for a whole fi le so the change happens in one commit? I see e.g. `test_positive_run_default_job_template_multiple_hosts_by_ip`.

Reply:
Discussed with @pondrejk and decided to remove it globally.